src/Native/configure.cmake

```diff
@@ -163,6 +163,17 @@ check_struct_has_member(
     "sys/select.h"
     HAVE_PRIVATE_FDS_BITS)
 
+check_cxx_source_compiles(
+    "
+    #include <sys/sendfile.h>
+    int main() { int i = sendfile(0, 0, 0, 0); }
+    "
+    HAVE_SENDFILE)
+
+check_function_exists(
+    fcopyfile
```

Comment:
I don't believe so, or at least I'm not aware of another function that might conflict with it such that we'd need to differentiate based on header.  Am I wrong?